build/fuses/fuses.json

```diff
@@ -0,0 +1,5 @@
+{
+  "_comment": "Modifying this file in any way (removing, adding or changing values) should result in the _version prop being incremented",
```

Comment:
trying to think of if there's a way we can make this work so that an older tool can (at least sometimes) successfully edit a newer exe. which would be useful because apps wouldn't have to constantly update their `@electron/fuses` dep in order to continue setting the same fuse.

If fuses were only ever added to the end (never removed or changed), and _version was just the number of flags, then the tool could just check that _version > the index of the flag it wants to change.

We could indicate removed flags with a value of `f` instead of `0`/`1`, then `@electron/fuses` could first read the flag to see if it's non-removed, and warn if it tries to set a flag that's been removed.

If we did that scheme, we'd probably also want an escape-hatch that would prevent an out-of-date fuser from modifying the exe, in case we change this fuse scheme. We could either add a 2nd version field (maybe rename `_version` to `_length` and use `_version` for this backwards-incompatible indicator), or we could regenerate the sentinel, though the latter would make the case of trying to edit the wrong binary indistinguishable from the version being wrong, so we'd get better error messages if we keep the sentinel stable.

And of course, `_length` can be automatically computed, so there'd be no need to write it in the JSON file directly :)